app/views/posts/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1> Showing posts for user with id:<%= @user_id %></h1>
\ No newline at end of file
+<%= render partial: "partials/user", locals: {user: @user} %>
+
+<section class="posts_section">
+  <h3> Numumber of Posts</h3>
+  <div class="posts_container">
+    <ul class="posts_list">
+    <% @posts.each do |post| %>
+      <div class="post">
+      <%= render partial: "partials/post", locals: {post: post} %>
+      <h4>Comments: </h4>
+      <ul class="comments_list">
+        <% post.recent_comments.each do |comment| %> 
```

Comment:
There is a naming issue with your code. In your models you have `latest_comments and latest_posts methods` however, in the views, you refer to recent_comments and recent_posts.

https://github.com/Harrynoqs/blog-app/blob/5fb0686843c88820931b085b6db1ccee2fbfc217/app/models/post.rb#L11-L13

https://github.com/Harrynoqs/blog-app/blob/5fb0686843c88820931b085b6db1ccee2fbfc217/app/models/user.rb#L6-L8

Kindly fix this as suggested by the previous reviewer.